### PR TITLE
Remove nameless todo

### PR DESCRIPTION
**Pull request by chunhtai**
These todos do not seem to be necessary. We would like to enforce a more strict todo format, and this may be problematic.

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [ ] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: dev/integration_tests/flutter_gallery/lib/demo/material/search_demo.dart [hunk 1]
```diff
@@ -211,7 +211,7 @@ class _SearchDemoSearchDelegate extends SearchDelegate<int?> {
           tooltip: 'Voice Search',
           icon: const Icon(Icons.mic),
           onPressed: () {
-            query = 'TODO: implement voice input';
+            query = 'Implement voice input';
```
Comment (chunhtai) [inline dev/integration_tests/flutter_gallery/lib/demo/material/search_demo.dart:214]: @goderbauer

File: packages/flutter_test/lib/src/test_compat.dart [hunk 2]
```diff
@@ -144,7 +144,7 @@ Future<void> _runSkippedTest(Suite suiteConfig, Test test, List<Group> parents,
 ///       // This test is especially slow on Windows.
 ///       'windows': Timeout.factor(2),
 ///       'browser': [
-///         Skip('TODO: add browser support'),
+///         Skip('add browser support'),
```
Comment (chunhtai) [inline packages/flutter_test/lib/src/test_compat.dart:147]: @jonahwilliams

File: packages/flutter_tools/test/integration.shard/test_data/deferred_components_project.dart [hunk 3]
```diff
@@ -186,7 +186,7 @@ class BasicDeferredComponentsConfig extends DeferredComponentsConfig {
       }
 
       defaultConfig {
-          // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
+          // Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
```
Comment (chunhtai) [inline packages/flutter_tools/test/integration.shard/test_data/deferred_components_project.dart:189]: @GaryQian

Comment (jonahwilliams) [reply packages/flutter_tools/test/integration.shard/test_data/deferred_components_project.dart:189]: this is output text that is for the developer

File: packages/flutter_test/lib/src/test_compat.dart (hunk 2 above)
Comment (jonahwilliams) [reply packages/flutter_test/lib/src/test_compat.dart:147]: this is a doc comment example code block

File: packages/flutter_test/lib/src/test_compat.dart [hunk 4]
```diff
@@ -144,7 +144,7 @@ Future<void> _runSkippedTest(Suite suiteConfig, Test test, List<Group> parents,
...
 ///         // This will be slow on browsers once it works on them.
 ///         Timeout.factor(2)
 ///       ]
```
Comment (Hixie) [inline packages/flutter_test/lib/src/test_compat.dart:150]: the timeouts in this sample make my heart hurt

**Comment by Hixie** [comment]
test-exempt: only affects comments and tests

File: packages/flutter_test/lib/src/test_compat.dart (hunk 2 above)
Comment (chunhtai) [reply packages/flutter_test/lib/src/test_compat.dart:147]: I understand this does not really matter to our todo tech-debts, but my question is do we really need a `TODO:` here? the sentence itself should be enough?

Comment (jonahwilliams) [reply packages/flutter_test/lib/src/test_compat.dart:147]: fair enough

File: packages/flutter_tools/test/integration.shard/test_data/deferred_components_project.dart (hunk 3 above)
Comment (chunhtai) [reply packages/flutter_tools/test/integration.shard/test_data/deferred_components_project.dart:189]: Same thing here, do we really need to use a todo if this is not meant for us?

Comment (jonahwilliams) [reply packages/flutter_tools/test/integration.shard/test_data/deferred_components_project.dart:189]: yes

**Comment by chunhtai** [comment]
All of the todos I changed do not matter to us. Since they are not for us, I think we can avoid using todo here to avoid adding weirdness to out code base. I am always open to suggestions though.

**Comment by jonahwilliams** [comment]
tech debt measures should only count "real" todos. todos in generated code or doc comment examples should not count

**Comment by chunhtai** [comment]
Our current tech debt calculator does not excludes todos in generated code and doc I believe, but that is something we can change.

GCDC =  generated code or doc code sample

My argument here is it useful to allow using TODO for GCDC?
I can think of the following options:
1. keep them as is and update tech debt scripts to take all corner cases into account.
2. All TODO need to have an assignee. You either don't use TODO in GCDC or add some flag to make the script to ignore it.

I am not sure if (1) is even possible unless we keep a list of generated file somewhere, and it may be error prompt.

**Comment by jonahwilliams** [comment]
Yeah, I understand that seems like kind of a pain. Since its just the one place we can re-phrase it, though I'm weary of making too many concessions to tech debt benchmarks that we could instead adjust.

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Comment by Hixie** [comment]
I'm not 100% convinced this was a win. Doesn't this make the developer experience worse? Before they could just search for TODO to find things they needed to update. Does it matter if it puts a floor on our arbitrary benchmark?

**Comment by chunhtai** [comment]
I though more about this, I feel we should add an ignore flag to ignore the todo format, so that something like this can still be allowed. It will also be easier for us to exclude these TODO in our tech debt calculation.

**Comment by Hixie** [comment]
> exclude these TODO in our tech debt calculation.

I mean, if you are really worried about the slight impact they have on those calculations, just apply a flat subtraction to the calculation and call it a day. I would definitely not recommend spending much time worrying about the exact number. If we're ever down so far that they are a significant fraction of our technical debt number then we can have a party and worry about it then. Or just have the party. :-)

**Pull request merged**